pajbot/modules/vip_refresh.py

```diff
@@ -37,14 +39,20 @@ def _update_vips(self):
         self.bot.privmsg("/vips")
 
     @staticmethod
-    def _parse_pubnotice_for_vips(msg_id, message):
+    def _parse_pubnotice_for_vips(self, msg_id, message):
         if msg_id == "no_vips":
             return []
 
         if msg_id == "vips_success":
             if message.startswith("The VIPs of this channel are: ") and message.endswith("."):
                 message = message[30:-1]  # 30 = length of the above "prefix", -1 removes the dot at the end
-                return message.split(", ")
+                vips = message.split(", ")
+                # Response of "vips_success" returns display names, which could contain non-ascii characters.
+                # For now, we filter these out since these cannot be resolved through helix, see https://github.com/pajbot/pajbot/issues/1772
+                filtered_vips = [vip for vip in vips if re.match(self.username_regex, vip)]
+                if len(vips) != len(filtered_vips):
+                    log.warning("Some non-ascii display names were filtered out from vips_success NOTICE response.")
```

Comment:
I'm not sure about this error message, but I decided I'd give at least some indication of working around this issue. Because, even with this merged in, there will still be some users not acknowledged as VIPs (the ones with non-ascii usernames), so I think it's good to show that something's not right.

Reply:
I'm fine having this here :+1: